api/src/main/java/io/minio/MinioRequestBody.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Minio Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2015 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.minio;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.RandomAccessFile;
+import java.nio.channels.Channels;
+
+import okhttp3.MediaType;
+import okhttp3.RequestBody;
+import okio.BufferedSink;
+import okio.Okio;
+
+/**
+ * RequestBody that wraps a single data object.
+ */
+class MinioRequestBody extends RequestBody {
+  private final String contentType;
+  private final Object data;
+  private final int len;
+
+  MinioRequestBody(final String contentType, final Object data, final int len) {
```

Comment:
Could you change class name to `HttpRequestBody` like `HttpResponse`?

Reply:
Done